<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serieFoliosEditor</class>
 <widget class="QWidget" name="serieFoliosEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>&amp;Núm de Aprobación:</string>
     </property>
     <property name="buddy">
      <cstring>editNumAprobacion</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="MibitLineEdit" name="editNumAprobacion"/>
   </item>
   <item row="1" column="2" rowspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Código de barras bidimensional (CBB):</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblCbb">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCambiarCbb">
       <property name="text">
        <string>Agregar CBB</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Fecha de Aprobación:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="firstDayOfWeek">
      <enum>Qt::Monday</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Folio Inicial:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="MibitLineEdit" name="editFolioInicial"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>Folio Final:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="MibitLineEdit" name="editFolioFinal"/>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="lblTotalFolios">
     <property name="text">
      <string>.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MibitLineEdit</class>
   <extends>QLineEdit</extends>
   <header>../../mibitWidgets/mibitlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>editNumAprobacion</tabstop>
  <tabstop>btnCambiarCbb</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
